source/Calamari.AzureScripting/Scripts/AzureContext.ps1

```diff
@@ -185,14 +185,14 @@ Execute-WithRetry{
                     Initialize-AzContext
                 }
                 elseif (Get-AzureRmModuleInstalled) {
-                    if($OctopusAzureRMIsDepreciated -like [Boolean]::FalseString) {
-                        Write-Warning "Azure Resource Manager powershell module has reached end-of-life; please authenticate using Azure CLI or the Az module, Octopus will prevent usage of the AzureRM module in 2024.3."
-                        Initialize-AzureRmContext    
-                    }
-                    else {
+                    if($OctopusAzureRMIsDepreciated -like [Boolean]::TrueString) {
```

Comment:
@rain-on I've inverted the check here for the sake of the tests. I can't see this changing any other behaviour but just wanted to raise this incase.